core/arch/arm/kernel/user_ta.c

```diff
@@ -621,21 +504,410 @@ TEE_Result tee_ta_register_ta_store(struct user_ta_store_ops *ops)
 	return TEE_SUCCESS;
 }
 
-TEE_Result tee_ta_init_user_ta_session(const TEE_UUID *uuid,
-			struct tee_ta_session *s)
+static char tolowercase(char c)
```

Comment:
I'd prefer implementing the standard `tolower()`.
See https://github.com/jenswi-linaro/optee_os/commit/8ca272d6820f9008b61254a991e3c04a6c3b5cf8

Reply:
Sure, can you make a PR?

Reply:
https://github.com/OP-TEE/optee_os/pull/2283